curriculum/challenges/english/25-front-end-development/quiz-javascript-functions/66edcc779993c0da6906dbb9.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is the correct syntax for a function declaration?
 
 #### --distractors--
 
-Placeholder distractor 1
+var myFunction = function() {};
```

Comment:
how come this is not correct?

Reply:
The question is for function declaration and declarations simply use function keyword followed by name and parameters enclosed in parentheses.    
The highlighted line is function expression which works in slight different fashion, the functionName is assigned to the `function()` which is keyword.    

Plus the functions are hoisted, expressions are not.

Reply:
you are right, my bad

Reply:
Oh that's okay. Thanks for reviewing.